### PR TITLE
docs: added missing ubuntu dependency (libsrtp2-dev)

### PR DESCRIPTION
**Pull request by muka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Uses [aiortc](https://github.com/aiortc/aiortc) as Python WebRTC provider. This
 
 On Debian/Ubuntu run:
 ```
-apt install libavdevice-dev libavfilter-dev libopus-dev libvpx-dev pkg-config
+apt install libavdevice-dev libavfilter-dev libopus-dev libvpx-dev pkg-config libsrtp2-dev
```
Comment (ivelin) [inline README.md:21]: @muka Are you sure we need this dependency? Since we only use data channel, I don't think rtp is required at this stage.

Comment (muka) [reply README.md:21]: This was required during setup build, if I recall correctly

Comment (ivelin) [reply README.md:21]: Yes, its listed in the aiortc install docs. However if you look closely, all these dependencies are related to WebRTC media and none is really required at runtime for webrtc datachannel.

In the future if we decide to provide video streams from the cameras to the PWA, these dependencies will come handy. But for now they are excess weight.

There is an [open issue for aiortc](https://github.com/aiortc/aiortc/issues/78) to allow a lightweight datachannel distro.

Comment (ivelin) [reply README.md:21]: I will accept the merge for simplicty sake. It is not a big delta relatively speaking. 

Although I'd still like to move to a datachannel only distro when available until we decide to offer real time streaming or calling from edge to UI.

**Pull request merged**